src/ui/public/vislib/visualizations/pie_chart.js

```diff
@@ -327,6 +328,54 @@ export function VislibVisualizationsPieChartProvider(Private) {
       }
     }
 
+    readableNumber(number) {
+      let tier = Math.log10(Math.abs(number)) / 3 | 0;
+      if(tier === 0) return number;
+      if(tier >= this.ABBREVIATIONS.length) tier = this.ABBREVIATIONS.length - 1;
+      const postfix = this.ABBREVIATIONS[tier];
+      const scale = 10 ** (tier * 3);
+      const scaled = number / scale;
+      const formatted = Number(scaled.toFixed(1));
+      return `${formatted}${postfix}`;
+    }
+
+    addDonutDescriptor(svg, width, height) {
+      const metricLabel = this.events.handler.data.data
+        .raw.columns.find(function (element) {
+          return element.categoryName === 'metric';
+        }).label;
+      const descriptor = svg.append('text')
+        .style('text-anchor', 'middle')
+        .attr('font-size', function (chartData) {
+          const dividend = Math.min(width, height);
+          const length = chartData.raw ? chartData.raw.columns.length : 0;
+          const divisor = Math.max(length, 4);
+          return (dividend / divisor) + 'px';
+        })
+        .attr('dy', '.35em')
+        .text((chartData) => {
+          return this.readableNumber(chartData.slices.sumOfChildren);
+        });
+      let tooltip;
```

Comment:
i suggest to replace the whole section below with: 
`descriptor.call(this.tooltip.render(`<table><tr><td>Total ${metricLabel}</td></tr></table>`));`

this way we let our tooltip service take care of showing/hiding and positioning of tooltips.

this will require some changes to the `ui/vis/components/tooltip/tooltip.js`:
- on line 132 you should add a parameter to the render method: `defaultHtml`
- on line 186 you could then use this to render the tooltip: 
```
        if (defaultHtml) {
          return render(defaultHtml);
        }
```

Reply:
Makes Sense, Will be on to it.